src/views/Advertisement/Advertisement.js

```diff
@@ -0,0 +1,40 @@
+import React, { PropTypes } from 'react'
+import cx from 'classnames'
+import {
+  customPropTypes,
+  getElementType,
+  META,
+} from '../../lib'
+
+/**
+*An ad displays third-party promotional content
+*/
+function Advertisement(props) {
+  const {
+    className,
+  } = props
+
+  const classes = cx(
+    'ui',
+    'ad',
+    className
+  )
+  const ElementType = getElementType(Advertisement, props)
+
+  return <ElementType className={classes} />
```

Comment:
Also, to get test working you need use there `getUnhandledProps` function and spread `rest` to `ElementType`.